### ISSUE TITLE
Define URL scheme that allows updating

### ISSUE THREAD

**Issue by cookeac** [adewg/ICAR#147]
Build on the example URL scheme in issue #18 and address the question about use of PATCH/PUT for fields in issue #140 .
Create a URL scheme that includes **POST** (new resources), **PUT** and **PATCH** (update resources)

**Comment by ahokkonen** [comment]
Within that task I think we also should define standardized **response** messages for POST/PUT requests - event id (for mapping between request items and response statuses), processions status, possible errors, etc...

**Comment by ahokkonen** [comment]
As discussed during our previous technical meeting here is a response model used for event POST requests in Nordic Data Exchange:

```json
[
  {
    "EventID": "string",
    "AdditionalObjectIDs": [
      {
        "ObjectID": "string",
        "ObjectIDType": 0,
        "ObjectHierarchyLevel": 0
      }
    ],
    "Status": 0,
    "Errors": [
      {
        "ErrorID": 0,
        "ErrorText": "string",
        "ErrorFieldName": "string",
        "ErrorSource": 0
      }
    ]
  }
]
```
As you can this response is an array of object with specific status fields for each posted event. 

- EventID - tracking identifier used to map response object to original event by its ID
- Status - procession status (separate code set used)
- Errors - detailed information about errors if procession failed
- AdditionalObjectIDs - additional definition for sub resources (for example progenies inside parent birth event etc.)

**Comment by cookeac** [comment]
Will be addressed for POST when #154  is resolved.
